lib/content-services/src/lib/search/components/search-filter-container/search-filter-container.component.html

```diff
@@ -10,7 +10,7 @@
             class="adf-filter-button"
             [attr.aria-label]="getTooltipTranslation(col?.title)"
             [matTooltip]="getTooltipTranslation(col?.title)">
-        <adf-icon value="adf:filter"
+        <adf-icon value="filter_list"
```

Comment:
Has this change been approved by someone from the design team? New icon looks different than the old one

Reply:
Yes, this change has been verified by Shane Maciak.